examples/tutorials/mnist_pytorch/model_def.py

```diff
@@ -61,9 +61,34 @@ def build_training_data_loader(self) -> DataLoader:
             self.data_downloaded = True
 
         train_data = data.get_dataset(self.download_directory, train=True)
-        return DataLoader(train_data, batch_size=self.context.get_per_slot_batch_size())
 
-    def build_validation_data_loader(self) -> DataLoader:
+        mode = "I like pain"
```

Comment:
I want to call out that this helps porting in some cases, but this does not solve the case where a user has a get_data function that returns a dataloader and just passing that into build_data_loader. Talking to @rb-determined-ai it doesn't seem possible.

Reply:
Right, with this change, they can do exactly that, but if it's wrong, it's wrong.  There's no way for us to modify their existing data loader to make it correct, we can only give the user to tools to take care of it themselves.

If they have a map dataset, they can still return a `det.pytorch.DataLoader` as before.